dune/stuff/functions/expression.hh

```diff
@@ -303,6 +340,21 @@ private:
       }
   } // ... build_gradients(...)
 
+  template< int rows, int cols >
+  void build_gradients(const std::string variable,
+                       Dune::FieldMatrix< std::string, rows, cols > jacobian)
+  {
+    assert(jacobian.rows == 0 || jacobian.rows >= dimRange);
```

Comment:
This is not how a `FieldMatrix` works. It has a static size of `rows` times `cols`, so no checks needed.